src/main/webapp/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+
+<title>ZoomTube</title>
+<link rel="stylesheet" href="style.css" />
+<script src="transcript.js"></script>
+<body>
+  <p>Hello world!</p>
+  <!-- TODO: Replace Button with a form that uses User Input. -->
+  <button type="button" onclick="sendPostToTranscript();">sendPostToTranscript</button>
+  <div id="transcript-container"></div>
+</body>
```

Comment:
is index.html just for testing?  Can this be merged with lecture-view.html?

Reply:
Yes, that is just for testing. Thank you for reminding me to move it over.
Done!